packages/headless/src/database/mongodb/common/modelOptions.ts

```diff
@@ -0,0 +1,14 @@
+export const modelOptionsFactory = () => {
+  return {
+    timestamps: true,
+    versionKey: false,
+    toJSON: {
+      transform: (_, ret) => {
+        delete ret._id;
+        delete ret.createdAt;
+        delete ret.updatedAt;
+        return ret;
+      },
+    },
```

Comment:
Seems good to use separate function to reduce code redundancy.
But for visibility we may skip this improvement. I've discussed with rezve vai. He also suggested to not overwrite toJSON options. and also he recommended to use the options in schema, rather than using from function response

Reply:
Here only the toJSON and toObject options provides the transform method to overwrite the response doc.
Would you please elaborate,
If we need the same transformation for all of the schema what will be the problem in visibility? and
What will be the problem if we are overwrite toJSON options?

I didn't find any better solution by which we can transform the response doc without using the transform method. Ans this also requires less code. That's why I've implemented in this way. 
It would be very helpful if you can provide an example how we can use the options in schema.